test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -379,6 +380,28 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
     });
   });
 
+  describe("POST Submit with url params tests", () => {
+
+    test(`redirects to the  ${config.CHECK_YOUR_ANSWERS_PAGE} page`, async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true); // For FEATURE_FLAG_ENABLE_REDIS_REMOVAL
+      mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_MOCK);
+      mockcheckEntityRequiresTrusts.mockReturnValueOnce(false);
+      const resp = await request(app)
+        .post(config.BENEFICIAL_OWNER_TYPE_SUBMIT_URL);
+
+      expect(resp.status).toEqual(302);
+      expect(resp.text).toContain(NEXT_PAGE_URL);
+      expect(resp.header.location).toEqual(NEXT_PAGE_URL);
+      expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(1);
+      expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(config.CHECK_YOUR_ANSWERS_WITH_PARAMS_URL);
+    });
+
+    /*
```

Comment:
Is this comment needed here? Or perhaps, rather, can most of the other 'submit' tests be included here too (now). Perhaps not those related to trusts but looks like other scenarios that should work, once this page is delivered.

Actually, some unit tests related to testing and I think @markpit might have spotted something related to validation, so maybe he'll add those tests under a new story?

Reply:
I can add the validation tests under https://companieshouse.atlassian.net/browse/ROE-2409 but in order to do so I'll have to add/pass the other BO/MO urls into the template besides the current BO Individual.

Reply:
I left the other 'submit' unit tests out as they all appeared to be validation related and this is just happy path. I can try and get them working here but not sure what value they'll be adding given the validation will be addressed in another ticket.

Reply:
👍 happy for these to be done by @markpit and that other ticket. Approving.